tests/dummy-app/processors/random.ts

```diff
@@ -0,0 +1,24 @@
+import Random from "../resources/random";
+import { OperationProcessor, HasId, Operation } from "../jsonapi-ts";
+import jsonApiErrors from "../../../src/errors/json-api-errors";
+
+const randomDataGenerator = {
+  number: () => ({ randomNumber: Math.random() }),
+  string: () => ({ randomString: parseInt(Math.random().toString().split(".")[1]).toString(16) }),
+  date: () => ({ randomDate: new Date(1602518929 + Math.random() * 10000000000000).toJSON() })
+}
+
+export default class RandomProcessor<ResourceT extends Random> extends OperationProcessor<ResourceT> {
+  static resourceClass = Random;
+
+  async get(op: Operation): Promise<HasId | HasId[]> {
```

Comment:
this `| HasId[]` is necessary or correct?, the implementation of the function below seems to only use one id, unless I'm reading this wrong...

Reply:
That's the signature for the `get` operation in the `OperationProcessor`:

https://github.com/kurierjs/kurier/blob/8c3f134dccdb1c868c545e9a054f1a97a13b7f1e/src/processors/operation-processor.ts#L123